### PR TITLE
Do not use hook AdminStatsModules alias, use displayAdminStatsModules & Bump version to 2.0.1

### PR DESCRIPTION
**Pull request by PrestaEdit**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Do not use hook AdminStatsModules alias, use displayAdminStatsModules
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | n/d
| Possible impacts? | n/d

### CODE REVIEW

File: upgrade/upgrade-2.0.1.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ */
+
+/**
+ * @param Module $module
+ *
+ * @return bool
+ */
+function upgrade_module_2_0_1$module)
```
Comment (PierreRambaud) [inline upgrade/upgrade-2.0.1.php:27]:
```suggestion
function upgrade_module_2_0_1($module)
```
Watch out, it's not the first time :p

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
A little change is needed

File: upgrade/upgrade-2.0.1.php (hunk 1 above)
Comment (PrestaEdit) [reply upgrade/upgrade-2.0.1.php:27]: Strange, I've take the previous PR where I think it was already commited. Sorry.

**Review by PierreRambaud — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Comment by atomiix** [comment]
Before the upgrade:
Capture d’écran 2022-02-03 à 11 27 45

After the upgrade:
Capture d’écran 2022-02-03 à 11 31 48

Everything is fine! QA ✔️

**Comment by atomiix** [comment]
Thanks @PrestaEdit

**Pull request merged**